src/Web/ClientApp/.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
+    "trailingComma": "none",
```

Comment:
If this is the rule enforcing the trailing comma after an enumeration, I don't think we should deactivate it